gradle.properties

```diff
@@ -15,3 +15,4 @@ POM_PACKAGING=jar
 POM_EMAIL=karol.wrotniak@droidsonroids.pl
 POM_INCEPTION_YEAR=2016
 POM_DEVELOPER_NAME=Karol Wrótniak
+android.useAndroidX=true
```

Comment:
Fixes: 
>Execution failed for task ':sample:processDebugAndroidTestManifest'.
> This project uses AndroidX dependencies, but the 'android.useAndroidX' property is not enabled. Set this property to true in the gradle.properties file and retry.
  The following AndroidX dependencies are detected: androidx.test:monitor:1.2.0, androidx.test:rules:1.2.0, androidx.test.ext:junit:1.1.1, androidx.test:core:1.2.0, androidx.lifecycle:lifecycle-common:2.0.0, androidx.test:runner:1.2.0, androidx.annotation:annotation:1.0.0